skrf/media/physicalLine.py

```diff
@@ -0,0 +1,177 @@
+
+
+'''
+.. module:: skrf.media.physicalLine
+
+========================================
+physicalLine (:mod:`skrf.media.physicalLine`)
+========================================
+
+A generic physical transmission line media.
+
+Represents a generic physical transmission line, defined by impedance,
+attenuation, relative permittivity and loss angle.
+
+Frequency invariant parameters or Djirdjevic/Svennson dispersion is provided. 
+
+
+
+'''
+from scipy.constants import  epsilon_0, c
+from numpy import real, imag, sqrt, ones, zeros, pi, log
+from .media import Media
+
+class PhysicalLine(Media):
```

Comment:
Perhaps we could come up with a more descriptive name?  physical line could equally well be used for  RLCG. i dont have any great ideas,  perhaps Z0AEp ?

Reply:
indeed, almost all the transmission lines are so-called physical lines, or it would be difficult to manufacture them.

What's about DefinedEpTandZ0? It would keep the spirit of DefinedGammaZ0.

Reply:
yeah good point!  that works

Reply:
Done in   7f6cc27